src/scenes/home/getInvolved/getInvolved.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import LinkButton from 'shared/components/linkButton/linkButton';
+import HeroBanner from 'shared/components/heroBanner/heroBanner';
+import banner1 from 'images/bannerimg1.jpg';
+import WhyGive from './whyGive/whyGive';
+import Volunteer from './volunteer/volunteer';
+import SuccessStories from './successStories/successStories';
+import styles from './getInvolved.css';
+
+const getInvolved = () => (
+  <div className={styles.page}>
+    <HeroBanner
+      imageSrc={banner1}
+      title="Get Involved"
+      subtitle="See the magic happen..."
+    >
+      <LinkButton
+        text="Donate"
+        theme="red"
+        link="https://opencollective.com/operationcode#support"
+        isExternal
+      />
+      <LinkButton text="Volunteer" theme="blue" link="/signup" />
```

Comment:
I don't think this LinkButton component needs to be nested here inside of the HeroBanner - I don't see any buttons anywhere?

Reply:
Nuts - that's something I lost track of. It's on the wireframes to have a button in the hero banner. I'll remove it for now, and I can create an issue to try to allow link buttons as possible child components in the hero banner component.